lib/xdrgen/output.rb

```diff
@@ -18,7 +18,7 @@ def relative_source_paths
     end
 
     def relative_source_path_sha256_hashes
-      relative_source_paths.map { |p| [p, Digest::SHA256.file(p).hexdigest] }.to_h
+      relative_source_paths.map { |p| [p, Digest::SHA256.hexdigest(File.read(p).gsub(/[[:space:]]/,''))] }.to_h
```

Comment:
Could we normalize the line endings instead of scrub whitespace from the whole files? Scrubbing whitespace makes it harder to debug and will cause files with space changes to have the same hash.

Reply:
Actually I think https://github.com/stellar/stellar-xdr-next/pull/32 is preferred over this.